notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -18,6 +18,16 @@
     "- Compare the performance of the original, converted and quantized models."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            input='input_ids[1 128],attention_mask[1 128],token_type_ids[1 128]',
</pre><p>no need in input and output specification in conevrt_model if there is no difference between onnx and it (previously input used to turn dynamic inputs to static</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/941/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>